src/components/Tags.js

```diff
@@ -0,0 +1,30 @@
+import XButton from "./XButton";
+import styles from "./Tags.module.css";
+
+const Tags = ({ tags, onRemove, className = "" }) => {
+  return (
+    <>
+      {tags.length ? (
+        <ul className={`${styles.tags} ${className}`}>
+          {tags.map((tag) => {
+            return (
+              <li key={tag} className={styles.tag}>
+                <span>#{tag}</span>
+                {onRemove && (
+                  <XButton
+                    className={styles.icon}
+                    onClick={(event) => onRemove(event, tag)}
+                  />
+                )}
+              </li>
+            );
+          })}
+        </ul>
+      ) : (
+        ""
+      )}
```

Comment:
P2:
이런경우 null을 반환하시는게 더 명시적일 것 같습니다.
```suggestion
      ) : null}
```